src/xbot/common/properties/RobotDatabaseStorage.java

```diff
@@ -2,7 +2,7 @@
 
 public class RobotDatabaseStorage extends DatabaseStorageBase {
 
-    private static String location = "/TeamDatabase";
+    private static String location = "/home/lvuser/TeamDatabase";
```

Comment:
- This should be `final`
- It feels wrong to hard-code the location of the database like this as an absolute path... not sure how we can fix that though.